account_invoice_supplierinfo_update/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        ALTER TABLE wizard_update_invoice_supplierinfo
+        DROP CONSTRAINT IF EXISTS wizard_update_invoice_supplierinfo_invoice_id_fkey
+        """,
+    )
+    # Prevent having records that weren't vacuumed as they could provoke a key error
+    # for the new constraint
+    openupgrade.logged_query(
+        env.cr, "DELETE FROM wizard_update_invoice_supplierinfo_line"
+    )
+    openupgrade.logged_query(env.cr, "DELETE FROM wizard_update_invoice_supplierinfo")
```

Comment:
Do `TRUNCATE wizard_update_invoice_supplierinfo CASCADE`, and prevent any problem with extensions.

Reply:
Done